README.md

```diff
@@ -33,7 +33,7 @@ Binary distributions of JDK Mission Control are provided by different downstream
 * Integrated (in-app) update site
 * Eclipse update site
 
-[http://jdk.java.net/jmc](http://jdk.java.net/jmc)
+[https://www.oracle.com/java/technologies/javase/products-jmc7-downloads.html](https://www.oracle.com/java/technologies/javase/products-jmc7-downloads.html)
```

Comment:
Are you sure you want the URL to be the jmc7 downloads? Do you have a top level download URL that will point to the latest always?